spring-aot/src/main/java/org/springframework/context/bootstrap/generator/bean/descriptor/BeanInstanceDescriptor.java

```diff
@@ -47,12 +48,15 @@
 
 	private final List<PropertyDescriptor> properties;
 
+	private final List<InitializationCallback> initializationCallbacks;
```

Comment:
Having looked at the end result, I think this should rather be a `MemberDescriptor<Method>`. This way we can move the code generation to the writer and the reflection configuration can be provided based on the descriptor.